src/Tribe/Admin/Settings.php

```diff
@@ -221,6 +222,9 @@ public function add_admin_pages() {
 			]
 		);
 
+		// Redirects users from any TEC setting to the Guided Setup page on first visit.
+		do_action( 'tec_redirect_guided_time_setup' );
```

Comment:
A suggestion for adding 'admin_' to the prefix, which feels more in-line with our naming standards.